examples/README.md

```diff
@@ -41,7 +41,6 @@ Windows support covers traces and metrics collection only, no logs will be colle
 image:
   otelcol:
     repository: quay.io/signalfx/splunk-otel-collector-windows
-logsEnabled: false
```

Comment:
I think we also have a documentation note about this limitation above, please remove it.

Reply:
Thanks! Removed that limitation note